Changes

```diff
@@ -78,6 +96,7 @@ Fixes
   - Fixed bug when deselecting a curve - the plug listing selection was not being updated
     to reflect the change.
 - PythonCommand : Prevented inadvertent modification of outer context (#3101).
+- Viewer : Prioritised update of objects being manipulated (#3144).
```

Comment:
Let's move this to the improvements section, and give it some prominence near the top, mentioning that it provides much more fluid updates than before. In the commits in that PR I also listed several breaking changes that need to be added to the breaking changes section.